### PR TITLE
Ert medic kit rework

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
This PR is designed to consolidate and update the kit of the medical ERT across all three levels. The changes made give the ERT a unique medical kit that replaces older kits and reduces the number of items in the backpack inventory. This includes removing all the existing first aid kits in the inventory of all three levels of ERT medical. These changes also move the magboots from the ERT Red inventory and equips them, as well as doing the same for the ERT Amber helmet. 

**Items within the new ERT Amber first-aid kit:**

- Advanced Health Analyzer - x 1
- Brute Auto-mender - x 1
- Burn Auto-mender - x 1
- Advanced Trauma Kit - x 1
- Advanced Burn Kit - x 1
- Pill Bottle – x 1
   - Salbutamol – x 6
   - Charcoal – x 6
   - Salicylic Acid – x 6
- Patch Pack – x 1
   - Small Burn Patch – x 5
   - Small Brute Patch – x 5

**Items withing the new ERT Red/Gamma first-aid kit:**

- Advanced Health Analyzer - x 1
- Dual Auto-mender - x 1
- Splint – x 1
- Advanced Trauma Kit - x 1
- Advanced Burn Kit - x 1
- Pill Bottle – x 1
   - Pentetic Acid – x 6
   - Iron/Saline-Glucose – x 6
   - Salicylic Acid – x 6
- Patch Pack – x 1
   - Perfluorodecalin – x 5
   - Burn Patch – x 5
   - Brute Patch – x 5

## Why It's Good For The Game
This PR is good because it makes the ERT medics kit more efficient and cleaner looking. This PR also serves to give ERT medics a special feel that should exist outside what is normally available to the player.
## Images of changes
**Amber/Red Original:**
amber_original
red_original

**Amber Changed:**
amber_changed
amber_changed_kit

**Red Changed (Gamma has same kit included):**
red_changed
red_changed_kit

## Testing
Equipped new gear for each medic type in testing environment. 
Tested medication and observed kit to check intended changes.

## Changelog
:cl:
del: exisiting first-aid and pill bottles located within ERT medic inventories
add: a new first-aid kit (ERT) and populates it, including a new patch pack and pill bottle
add: a new first-aid kit (ERT Amber) and populates it, including a new patch pack and pill bottle
tweak: ERT Red now spawns with their magboots equipped
tweak: ERT Amber now spawns with their helmet equipped
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -213,6 +213,40 @@
...
+	new /obj/item/healthanalyzer/advanced(src)
+	new /obj/item/stack/medical/splint(src)
+	new /obj/item/reagent_containers/applicator/dual(src)
+	new /obj/item/stack/medical/ointment/advanced(src)
+	new /obj/item/stack/medical/bruise_pack/advanced(src)
+	new /obj/item/storage/pill_bottle/ert(src)
+	new /obj/item/storage/pill_bottle/patch_pack/ert/(src)
+
+/obj/item/storage/firstaid/ert/empty/populate_contents()
+	return
```
Comment (Coolrune206) [inline code/game/objects/items/weapons/storage/firstaid.dm:231]: Is this part of the code necessary? I don't think the empty variant of the ERT specific medkit would be used anywhere.

Comment (Coolrune206) [reply code/game/objects/items/weapons/storage/firstaid.dm:231]: Ditto for the amber

Comment (Burzah) [reply code/game/objects/items/weapons/storage/firstaid.dm:231]: It probably would not be necessary, but I was following similar kit builds within the code. The only thing I could think of its use is if an admin spawned in an empty kit to do some type of custom inventory population given for an event or specific situation.

Comment (Burzah) [reply code/game/objects/items/weapons/storage/firstaid.dm]: Removed the empties. Tested to see if the kits, pill bottles, and patch packs still function. Resolved with [e4e67ed](https://github.com/ParadiseSS13/Paradise/pull/20561/commits/e4e67ed1e4c82c11e5f8893ded02a79acea61102)

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 2]
```diff
@@ -213,6 +213,34 @@
...
+
+/obj/item/storage/firstaid/ert_amber/populate_contents()
+	new /obj/item/healthanalyzer/advanced(src)
+	new /obj/item/reagent_containers/applicator/brute(src)
+	new /obj/item/reagent_containers/applicator/burn(src)
+	new /obj/item/stack/medical/bruise_pack/advanced(src)
+	new /obj/item/stack/medical/ointment/advanced(src)
+	new /obj/item/storage/pill_bottle/ert_amber(src)
+	new /obj/item/storage/pill_bottle/patch_pack/ert_amber/(src)
+
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/firstaid.dm:243]:
```suggestion
	new /obj/item/storage/pill_bottle/ert_amber(src)
	new /obj/item/storage/pill_bottle/patch_pack/ert_amber(src)

```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 3]
```diff
@@ -213,6 +213,34 @@
...
+/obj/item/storage/firstaid/ert/populate_contents()
+	new /obj/item/healthanalyzer/advanced(src)
+	new /obj/item/stack/medical/splint(src)
+	new /obj/item/reagent_containers/applicator/dual(src)
+	new /obj/item/stack/medical/ointment/advanced(src)
+	new /obj/item/stack/medical/bruise_pack/advanced(src)
+	new /obj/item/storage/pill_bottle/ert(src)
+	new /obj/item/storage/pill_bottle/patch_pack/ert/(src)
+
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/firstaid.dm:229]:
```suggestion
	new /obj/item/storage/pill_bottle/ert(src)
	new /obj/item/storage/pill_bottle/patch_pack/ert(src)

```

**Review by farie82 — APPROVED** [review]

**Pull request merged**